packages/graphql-transformer-core/src/TransformerContext.ts

```diff
@@ -167,6 +289,70 @@ export default class TransformerContext {
         }
     }
 
+    /**
+     * Extends the context query object with additional fields.
+     * If the customer uses a name other than 'Query' this will proxy to the
+     * correct type.
+     * @param fields The fields to add the query type.
+     */
+    public addQueryFields(fields: FieldDefinitionNode[]) {
+        const schemaNode = this.getSchema();
+        const queryTypeOperation = schemaNode.operationTypes.find((op: OperationTypeDefinitionNode) => op.operation === 'query');
+        const queryTypeName = queryTypeOperation && queryTypeOperation.type && queryTypeOperation.type.name && queryTypeOperation.type.name.value ?
+            queryTypeOperation.type.name.value : undefined;
```

Comment:
It looks like you're duplicating the work from getQueryTypeName() here. Shouldn't you just call that function?